R/common-error-checking-utils.R

```diff
@@ -0,0 +1,24 @@
+#' @title 
+#' Boolean. Tests whether predictedCol is Y/N
+#' 
+#' @description Returns a logical, TRUE or FALSE, depending on what is
```

Comment:
Description shoudl be spelled out further to say how the function behaves. "Returns TRUE if a column contains only Y/N values. NA values are ignored."